docker/docker-compose.connect-popup-test.yml

```diff
@@ -18,7 +18,7 @@ services:
     network_mode: service:trezor-user-env-unix
 
   test-run:
-    image: mcr.microsoft.com/playwright:latest
+    image: mcr.microsoft.com/playwright:v1.39.0-jammy
```

Comment:
what specifically is this needed for?

Reply:
This is just for development. 
But there is some times when locally the latest version of Playwright was giving me an error.